packages/eslint-plugin/src/rules/prefer-nullish-coalescing.ts

```diff
@@ -1,6 +1,7 @@
 import type { TSESLint, TSESTree } from '@typescript-eslint/utils';
 import { AST_NODE_TYPES, AST_TOKEN_TYPES } from '@typescript-eslint/utils';
 import * as tsutils from 'tsutils';
+import { isTypeFlagSet, isUnionOrIntersectionType } from 'tsutils';
```

Comment:
Nit: we already `import * as tsutils` in this file, any particular reason to add a second import form?